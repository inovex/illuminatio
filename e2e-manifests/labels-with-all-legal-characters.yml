apiVersion: v1
kind: Namespace
metadata:
  name: labels-with-all-legal-characters
  labels:
    illuminatio-e2e: labels-with-all-legal-characters
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test
  namespace: labels-with-all-legal-characters
spec:
  selector:
    matchLabels:
      test.io/test-123_XYZ: test_456-123.ABC
  template:
    metadata:
      labels:
        test.io/test-123_XYZ: test_456-123.ABC
    spec:
      containers:
      - image: nginx
        name: test
        ports:
        - containerPort: 80
          protocol: TCP
---
apiVersion: v1
kind: Service
metadata:
  name: test
  namespace: labels-with-all-legal-characters
spec:
  type: ClusterIP
  ports:
  - port: 80
    protocol: TCP
    targetPort: 80
  selector:
     test.io/test-123_XYZ: test_456-123.ABC
---
kind: NetworkPolicy
apiVersion: networking.k8s.io/v1
metadata:
  name: api-allow
  namespace: labels-with-all-legal-characters
spec:
  podSelector:
    matchLabels:
      test.io/test-123_XYZ: test_456-123.ABC
  ingress:
  - from:
    - podSelector:
        matchLabels:
          test.io/test-123_XYZ: test_456-123.ABC
