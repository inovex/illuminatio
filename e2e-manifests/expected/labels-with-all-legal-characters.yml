illuminatio-inverted-labels-with-all-legal-characters:test.io/test-123_XYZ=test_456-123.ABC:
  labels-with-all-legal-characters:test.io/test-123_XYZ=test_456-123.ABC:
    -*:
      success: true
labels-with-all-legal-characters:illuminatio-inverted-test.io/test-123_XYZ=test_456-123.ABC:
  labels-with-all-legal-characters:test.io/test-123_XYZ=test_456-123.ABC:
    -*:
      success: true
illuminatio-inverted-labels-with-all-legal-characters:illuminatio-inverted-test.io/test-123_XYZ=test_456-123.ABC:
  labels-with-all-legal-characters:test.io/test-123_XYZ=test_456-123.ABC:
    -*:
      success: true
labels-with-all-legal-characters:test.io/test-123_XYZ=test_456-123.ABC:
  labels-with-all-legal-characters:test.io/test-123_XYZ=test_456-123.ABC:
    "*":
      success: true
